styles/style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  margin: 0px;
+  font-family: 'Merriweather', serif;
+  width: 100%
+}
+
+/* nav bar has three columns */
+nav ul {
+  display: grid;
+  grid-template-columns: 70% 15% 15%;
+}
+
+/* change background color and fix position of nav */
+.main-container nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background: slategray;
+  border-bottom: #333
+}
+
+/* align the welcome sub and button */
+.welcome {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10%
+}
+
+.welcome h1, sub {
+  font-size: 3em;
+  padding: 5px
+}
+
+/* make button black and green on hover */
+.button {
+  display: flex;
+  font-size: 1em;
+  padding: 20px;
+  margin-bottom: 20px;
+  text-decoration: none;
+  background-color: black;
+  border-radius: 8px;
+  color: white;
+  align-items: center;
+  justify-content: center;
+}
+
+.welcome div {
+  border: 2px solid transparent;
+  border-radius: 8px;
+}
+
+.button:hover {
+  background-color: greenyellow;
+  color: black;
+  border: 2px solid transparent;
+  border-radius: 8px;
+
+}
+
+/* align quotes to center and author to bottom right */
+.quote {
+  padding: 1em;
+  border-top: 1px solid #333;
+  border-bottom: 1px solid #333;
+}
+
+.quote {
+  display: flex;
+  flex-direction: column;
+  font-style: italic;
+}
+
+.quote p {
+  align-self: flex-end;
+  margin-right: 5%
+}
+
+blockquote {
```

Comment:
It is possible to add quotations to this section by incorporating something like:

blockquote:before {
  content: open-quote;
}

blockquote:after {
  content: close-quote;
}